.changeset/cool-books-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/lexicon": minor
+---
+
+Add support for `permission-set` in lexicon documents
```

Comment:
What made this minor vs. patch (pre-v1)?  Or nothing in particular, just opting for minor as a marker?

Reply:
It's minor because now, `lexicon.get('nsid').defs.main` can return a `LexPermissionSet` which was not the case before. So any code doing things like: 

```ts
const { type } = lexicon.get('nsid').defs.main
switch (type) {
  case "record":
  case "...":
    console.log('all good');
    break
  default:
    throw new Error(`unexpected type ${type}`)
}

```

now has to account for that possibility